### DISCUSSION TITLE
Could the external antenna with the 3.5mm headphone plug enhance the wireless relay signal?

### DISCUSSION THREAD

**Discussion by 1457384613gh** [Mygod/VPNHotspot#282] [Q&A]
Could the external antenna with the 3.5mm headphone plug enhance the wireless relay signal?

![Screenshot_2021-09-23-08-32-34-2097599885.png](https://user-images.githubusercontent.com/69227436/134791327-8738f721-0d67-46ee-8aeb-5f4005e59c84.png)

**Comment by Mygod** [comment]
I don't think so. You can try though.
